connector/spanmetricsconnector/connector.go

```diff
@@ -127,19 +127,30 @@ func newConnector(logger *zap.Logger, config component.Config, ticker *clock.Tic
 	}, nil
 }
 
-func (p *connectorImp) initHistogramMetrics() metrics.HistogramMetrics {
-	cfg := p.config
+func initHistogramMetrics(cfg *Config) metrics.HistogramMetrics {
 	if cfg.Histogram.Exponential != nil {
 		maxSize := structure.DefaultMaxSize
 		if cfg.Histogram.Exponential.MaxSize != 0 {
 			maxSize = cfg.Histogram.Exponential.MaxSize
 		}
 		return metrics.NewExponentialHistogramMetrics(maxSize)
 	}
-	bounds := defaultHistogramBucketsMs
+
+	var bounds []float64
 	if cfg.Histogram.Explicit != nil && cfg.Histogram.Explicit.Buckets != nil {
 		bounds = durationsToUnits(cfg.Histogram.Explicit.Buckets, unitDivider(cfg.Histogram.Unit))
+	} else {
+		switch cfg.Histogram.Unit {
+		case metrics.Milliseconds:
+			bounds = defaultHistogramBucketsMs
+		case metrics.Seconds:
+			bounds = make([]float64, len(defaultHistogramBucketsMs))
+			for i, v := range defaultHistogramBucketsMs {
+				bounds[i] = v / float64(time.Second.Milliseconds())
```

Comment:
Did you consider the suggestion from: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21797 of using the bucket boundaries prescribed for `http.server.duration` suggested in: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#http-server?

Reply:
Sure thing, makes sense 👍🏼